### DISCUSSION TITLE
How to make Accordion slide-up?

### DISCUSSION THREAD

**Discussion by mationai** [mantinedev/mantine#2734] [Q&A] [answered]
Want to make an input area where one a single input field shows, with a trigger than unveils the past inputs. Pretty much what Accordion does except the reveal and transition direction. Is there any way to make this work?

**Comment by zhulien-ivanov** [comment]
https://mantine.dev/core/collapse/

**Reply by mationai** [reply]
Thank you. Saw this after posting and this is what I ended up using. But going to mark rtivital's as answer since his answer reflects more of the question.

**Comment by rtivital** [comment, marked as answer]
I did not really get the idea, but you can change Accordion.Panel and Accordion.Control order – https://codesandbox.io/s/heuristic-poitras-68ndmt?file=/src/App.tsx

**Reply by mationai** [reply]
Nice. Didn't know they can be reversed.
